documentation/3_Layers_of_the_Reference_Architecture_Model/3_2_Functional_Layer/3_2_FunctionalLayer.md

```diff
@@ -137,16 +137,19 @@ given in the [Process Layer](../3_4_Process_Layer/3_4_Process_Layer.md).
 
 #### Vocabularies ####
 
-To create and structure metadata, the operator of a Connector may use
-vocabularies. In doing so, an operator of a Connector can use existing
-vocabularies, create own vocabularies, or work with other operators on
-new vocabularies provided by vocabulary hubs. Vocabulary hubs are
-central servers that store vocabularies and enable collaboration.
-Collaboration may comprise search, selection, matching, updating,
-requests for changes, version management, deletion, duplicate
-identification, and unused vocabularies. Vocabulary hubs need to be
-managed. More information about vocabularies is given in the
-[Information Layer](../3_3_Information_Layer/3_3_InformationLayer.md).
+To create and structure metadata, the data provider in a data space may use vocabularies to be able to tell potential consumers of his data assets the semantics of the various data elements in the data assets at a universal location in a standard electronic format.
+With help of vocabularies the data consumer has possibility to easily understand of the semantics of the various data elements in an offered data asset before downloading it and verify the data asset afterwards.
+This universal location (Vocabulary Hub) is an IDS component (centralized server or decentralized network), that stores vocabularies and enables collaboration between participants to harmonise single vocabularies and create common set of harmonised vocabulary standards for the given data space.
+A set of functional requirements for this component is defined further:
+
+* The technical interface between the Vocabulary Hub and the data space infrastructure shall be based on the IDS compatible Connector.
+* The Vocabulary Hub shall have a browser-based user interface and API, which allows to visualize and to browsed vocabularies in a human user-friendly way.
+* The Vocabulary Hub shall provide creating, selection, editing (inserting, updating, changing, deleting, matching, version management), read- and search functionalities and support SPARQL queries.
+* Vocabularies expressed in RDF shall by syntactically compatible with OWL in order to enable processing with more expressive semantics
```

Comment:
- can this be generalized or is it only RDF and OWL?, something about in which formats vocabularies can be expressed and what are the compatibility requirements?
- can this be tied to Andreas' comment during the workshop?